main/mySpace/myStudents.php

```diff
@@ -2057,8 +2057,21 @@ function ($hookHeader) {
     $form->display();
 }
 
+$filter_messages = api_get_configuration_value('filter_interactivity_messages') && !empty($sessionId);
+
+if ($filter_messages) {
+    $session_info = api_get_session_info($sessionId);
+    $coach_access_start_date = $session_info['coach_access_start_date'];
```

Comment:
- Use camelCase
- Validate that the session exists 
- Set the default values of $coach_access_start_date/ $coach_access_end_date to null.

Reply:
Fixed!